### DISCUSSION TITLE
If GraphQL error is thrown we should have config option which still commits whatever is generated

### DISCUSSION THREAD

**Discussion by saihaj** [dotansimha/graphql-code-generator#8830] [Ideas]
Today if GraphQL error is encountered nothing generated is committed. 

I think the simple option is just committing the last valid generated type. But more advanced option is we skip the invalid parts and generated whatever we can.

**Comment by ZakRabe** [comment]
Has anything new happened for this feature? 

I thought `skipDocumentsValidation.skipValidationAgainstSchema` would accomplish this, but code generation is still stopped in case of invalid documents.
